

const throwErrorNow = (message) =>{
    throw new Error(message)
}

export default throwErrorNow